src/python/WMCore/ReqMgr/Web/ReqMgrService.py

```diff
@@ -211,6 +211,27 @@ def __init__(self, app, config, mount):
         identifier = cdict.get("log_reporter", "reqmgr2")
         self.logdb = LogDB(centralurl, identifier)
 
+        # local team cache which will request data from wmstats
+        self.wmstatsurl = config.reqmgr.wmstats_url
```

Comment:
Alan,
the problem is that config is not a pure python dict. It is WMCore object.
I didn't inspect if it supports get method and I provide a safe way
to fetch values from config. Of course if config would be a python
dict I'll use the way you described.
Valentin.

On  0, Alan Malta Rodrigues <notifications@github.com> wrote:
> > @@ -211,6 +211,27 @@ def __init__(self, app, config, mount):
> >          identifier = cdict.get("log_reporter", "reqmgr2")
> >          self.logdb = LogDB(centralurl, identifier)
> >  
> > +        # local team cache which will request data from wmstats
> > +        self.wmstatsurl = config.reqmgr.wmstats_url
> 
> How about keeping the same style as in the previous lines, e.g.:
> self.wmstatsurl = cdict.get('wmstats_url', '')
> 
> I'd also get rid of the if in the line below, we simply assume it will be in the configuration file.
> Otherwise, we better than check whether all the previous URLs are also available in the config file.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/dmwm/WMCore/pull/6681/files#r55367229